features/step_definitions/logging.rb

```diff
@@ -0,0 +1,128 @@
+### logging related step definitions
+# for 4.x we default logging installation is via OLM only
+
+Given /^logging service has been installed successfully$/ do
+  ensure_destructive_tagged
+
+  step %Q/I switch to cluster admin pseudo user/
+  step %Q/I use the "openshift-marketplace" project/
+  # first check packagemanifest exists for cluster-logging and elasticsearch-operator
+  required_packagemanifests = ['cluster-logging', 'elasticsearch-operator']
+  required_packagemanifests.each do |pm_name|
+    raise "Required packagemanifest #{pm_name} no found!" unless package_manifest(pm_name).exists?
+  end
+  # Create namespace
+  unless project('openshift-logging').exists?
+    # oc create -f https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/OCP-21311/namespace.yaml
+    namespace_yaml = "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/OCP-21311/namespace.yaml"
+    @result = admin.cli_exec(:create, f: namespace_yaml)
+    raise "Error creating namespace" unless @result[:success]
+  end
+  logging_ns = "openshift-logging"
+  operators_ns = "openshift-operators"
+
+  # project('openshift-logging')
+  # Create operator group in `openshift-logging` namespace
+  operator_group_yaml = "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/OCP-21311/operator-group.yaml"
+  @result = admin.cli_exec(:create, f: operator_group_yaml, n: logging_ns)
+  raise "Error creating namespace" unless @result[:success]
```

Comment:
maybe another error message

Reply:
what do you mean?  A more explicit error message with the name of the namespace?

Reply:
this error message is same as line 19 but looks like we are not creating a namespace but a group or something

Reply:
oh, yeah, copy and paste error...it should say `operatorgroup` .  Good catch!

Reply:
I still see `namespace` here.